django_cte/raw.py

```diff
@@ -19,15 +19,17 @@ def get_source_expressions(self):
             return []
 
     class raw_cte_compiler(object):
-        @staticmethod
-        def as_sql():
+        def as_sql(self, ):
```

Comment:
Why was this changed? I don't think it's necessary, but if it is please change to `def as_sql(self):`